custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -63,13 +64,14 @@ def _get_credentials(self, repeat_record):
 
     def _base_properties(self, repeat_record):
         username, password = self._get_credentials(repeat_record)
+        server_ip = socket.gethostbyname(socket.gethostname())
```

Comment:
i checked this on softlayer proxy machine and it returned an IP address. I think it should then work from celery machine as well where it would get triggered.
If it raises an exception it would be logged in the repeat record and i can ensure that it works as expected for first occurrence.
Its not a critical info but i am guessing we should not hardcode it.